pkg/volume/volume.go

```diff
@@ -33,6 +33,9 @@ type Volume interface {
 
 	// MetricsProvider embeds methods for exposing metrics (e.g. used,available space).
 	MetricsProvider
+
+	// GetPropagationMode returns the propagation mode the volume is mounted to.
+	GetPropagationMode() api.PropagationMode
```

Comment:
@liyimeng we're still talking about how to export the api in a right way. This method is just my personal proposal. If we put propagation mode in the `Attributes`, we also need to update the method `GetAttributes()` in every volume source. And thanks for your advice. :)